site/documentation/content/api/credentials/index.md

```diff
@@ -302,6 +302,39 @@ Example:
 | *auth-id*        | *yes*     | *string*   | The subject DN of the client certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). |
 
 {{% notice note %}}
-The example above does not contain any of the `not-before`, `not-after` and `enabled` properties. The `not-before` and `not-after` properties should be omitted if the validity period is the same as the period indicated by the client certificate's corresponding properties. It is still necessary to provide a (empty) JSON object in the *secrets* array, though.
+The example above does not contain any of the `not-before`, `not-after` and `enabled` properties. The `not-before`
+and `not-after` properties should be omitted if the validity period is the same as the period indicated by the client
+certificate's corresponding properties. It is still necessary to provide a (empty) JSON object in the *secrets* array,
+though.
 {{% /notice %}}
 
+### Asymmetric Key
```

Comment:
FMPOV we are only interested in the public key here, amended with some (optional) validity period. We have been thinking about adding a RawPublicKey credential type in the past already which covers exactly that. I would therefore like to use this opportunity to (finally :-)) introduce the *RawPublicKey* type into Hono and use *RawPublicKey* instead of *Asymmetric Key*.

Reply:
ok sounds good